src/style.scss

```diff
@@ -1,5 +1,41 @@
-$bg-colour: #ccc;
+$bg-colour: white;
+$border-color: #c1c1c1;
+$global-padding: 10px;
+$global-font: 'Roboto', sans-serif;
 
 body {
 	background-color: $bg-colour;
+	font-family: $global-font;
+}
+
+.c-vehicle-list {
+	display: grid;
```

Comment:
Grid supported by IE11+, Firefox, Chrome, Safari and Edge. I'd assume this is safe to use, rather than using a CSS framework or my own ugly implementation.

I'm a huge fan of simple code.

Reply:
https://caniuse.com/#feat=css-grid